src/main/java/org/intellij/sdk/codesync/server/CodeSyncReactivateAccountServer.java

```diff
@@ -0,0 +1,73 @@
+package org.intellij.sdk.codesync.server;
+
+import org.apache.http.client.utils.URIBuilder;
+import org.eclipse.jetty.server.Connector;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.ServerConnector;
+import org.eclipse.jetty.servlet.ServletHandler;
+import org.eclipse.jetty.util.thread.QueuedThreadPool;
+import org.intellij.sdk.codesync.CodeSyncLogger;
+import org.intellij.sdk.codesync.commands.Command;
+import org.intellij.sdk.codesync.server.servlets.ReactivateAccountHandler;
+
+import java.net.URISyntaxException;
+import java.util.LinkedList;
+import java.util.Queue;
+
+import static org.intellij.sdk.codesync.Constants.*;
+
+
+public class CodeSyncReactivateAccountServer {
+    private static CodeSyncReactivateAccountServer singletonInstance;
+    private Server server;
+    private static final Queue<Command> commandQueue = new LinkedList<>();
+
+    private CodeSyncReactivateAccountServer() throws Exception {
+        start();
+    }
+
+    public static CodeSyncReactivateAccountServer getInstance() throws Exception {
+        if (singletonInstance == null) {
+            singletonInstance = new CodeSyncReactivateAccountServer();
+        }
+        return singletonInstance;
+    }
+
+    public void start() throws Exception {
+        int maxThreads = 20;
+        int minThreads = 1;
+        int idleTimeout = 120;
+
+        QueuedThreadPool threadPool = new QueuedThreadPool(maxThreads, minThreads, idleTimeout);
+
+        server = new Server(threadPool);
+        ServerConnector connector = new ServerConnector(server);
+        server.setConnectors(new Connector[] { connector });
+
+        ServletHandler servletHandler = new ServletHandler();
+        server.setHandler(servletHandler);
+
+        servletHandler.addServletWithMapping(ReactivateAccountHandler.class, "/reactivate-callback");
+
+        server.start();
```

Comment:
Can we use a new endpoint in the existing server we use for Login Callback?
Also, maybe I missed, seems like we are using API to reactivate-account instead of going to webapp.

Reply:
I tried combining the 2 servers into 1 common but faced some issues. I think it would be better to add a ticket to refactor and test this.

Reply:
Sure, we can look into this later 👍